### DISCUSSION TITLE
I don't know who came up with the idea, ...name... Thaoma 2D

### DISCUSSION THREAD

**Discussion by ghost** [tahoma2d/tahoma2d#1013] [Ideas]
To the Great Team...

I don't want to offend anyone but every time I open the program... I get a shiver down my spine... Tahoma... as far as I know it's a type of font that I happen to like and use often... no problem at all ...

But naming an animation program of this caliber that I consider a piece of art is such a ridiculous name... barrr. not to mention the logo.....barrr..

With a lot of animators and creatives around the world using this program, you should consider running a contest for the new name and logo...

A name that has something to do with animation... easy to look up in search engines.

I have a career as an animator and director for over 40 years.. and in July 2022 I came to across the existence of this program for 2D animation, I just discovered it on a forum where they talk about OpenToonz Toonz, Moho and others...

So let's get to work...

Thanks to the development team that continues day by day to contribute to better tools where creativity has no limits...

Thank you all

**Comment by beeheemooth** [comment]
As for the name of the program, I have nothing against it, but the letter T always reminds me of the gallows for some reason :).

**Comment by artisteacher** [comment]
Well T2D definitely wasn't named after the font - I think it had to do with a moose? Hence the mascot.  Personally, I don't have a problem with it. I think it might be a little late to change the name at this point. However, people should always feel free to propose alternative logos. John might not choose to change it, but you never know until you try. Tbh, the angle of the T bothers me a bit...

**Comment by manongjohn** [comment]
Hehe...originally it was just named Tahoma, by the original developer who forked OT.  It was named after a moose in Canada zoo.  Later, "2D" was added to the name to help separate it from the font of the same name and kind of give it some idea of what the application is for. 

Personally, I am not particularly fond of the name, the mascot, logo and even the branding color choices.  When I took it over, T2D was just starting to stand on it's own and I didn't want to confuse users with a name change as well as having to change everything (i.e. websites, on-line manuals, repos, discords, redits, etc) around.  In retrospect, maybe I should have...

I won't rule out a complete rebranding in the future, but the conditions need to be right for me to want to do that.
